frontend/src/pages/_document.tsx

```diff
@@ -1,15 +1,29 @@
 import React from 'react'
+import getConfig from 'next/config'
 import { ServerStyleSheets } from '@material-ui/core/styles'
 import Document, { Html, Head, Main, NextScript } from 'next/document'
 
 import theme from 'common/theme'
 import FaviconMetadata from 'components/brand/FaviconMetadata'
 
+const GTM_ID = getConfig().publicRuntimeConfig.GTM_ID
+
 export default class CustomDocument extends Document {
   render() {
     return (
       <Html lang="en">
         <Head>
+          {/* Google Tag Manager */}
```

Comment:
I don't get why we have this placed in the document as the module we're using `react-gtm` is doing exactly this job

https://github.com/alinemorelli/react-gtm/blob/b1257405a82e900c174032a773a8ba70a8628081/src/Snippets.js#L12-L22

When we call `initialize` this is what is injected in our `<Head>` dynamically, so we don't have to do it manually.

Reply:
Looks like you are right. I should have read the `react-gtm-module` documentation :)